### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by jbrime15**
In this project, I have successfully displayed Hello microverse as described in the task URL below:

https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md

### CODE REVIEW

**Comment by jbrime15** [comment]
This project demonstrates how to use gitignore and Linters.

File: README.md [hunk 1]
```diff
@@ -1 +1,77 @@
-# Hello-Microverse-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+Hello-Microverse
+
+> One paragraph statement about the project.
+
+![screenshot](./app_screenshot.png)
```
Comment (Deepakdanger) [inline README.md]: This screenshot is not displaying. Kindly fix this 🔧

File: README.md [hunk 2]
```diff
@@ -1 +1,77 @@
...
+
+## Built With
+
+- Major languages (HTML and CSS)
+- Frameworks
+- Technologies used
+
+## Live Demos
+
+[Live Demo Link](https://livedemo.com)
```
Comment (Deepakdanger) [inline README.md]: This live demo link is broken. Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

File: README.md [hunk 3]
```diff
@@ -1 +1,77 @@
...
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (Deepakdanger) [inline README.md]: It is good to have well descriptive author information. Kindly change it

File: README.md [hunk 4]
```diff
@@ -1 +1,77 @@
...
+
+👤 **Author2**
```
Comment (Deepakdanger) [inline README.md]: Here `Author2` is displaying that is taking unnecessary attention, Please remove it.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../Hello-Microverse-/styles.css">
```
Comment (Deepakdanger) [inline index.html]: Your CSS style is not displaying in your index.html. There is some issue with the CSS file address, Kindly fix this

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @jbrime15, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
## Hi @jbrime15, 

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️ 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mre55 — COMMENTED** [review]
Hi @jbrime15, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :negative_squared_cross_mark:

You have submitted a project, that has been approved already. Kindly create a new branch from the main and copy what you have to the new branch and submit a new pull request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Pull request merged**